charts/rucio-daemons/templates/conveyor-finisher.yaml

```diff
@@ -1,4 +1,18 @@
 {{- if gt .Values.conveyorFinisherCount 0.0 -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "rucio.fullname" . }}.config.conveyor-finisher
+  labels:
+    app: {{ template "rucio.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rucio.config.common.json: {{ mustToPrettyJson .Values.config | b64enc | quote }}
+  rucio.config.component.json: {{ .Values.conveyorFinisher.config | default dict | mustToPrettyJson | b64enc | quote }}
```

Comment:
You are right, will update the PR to have the same syntax in both places and accept empty "common" configs. The difference is because I initially did it for "common". After that I realized that "component" configuration can be empty, so changed it to accept empty configs. 
But it's better to have the same behavior for both.